tests/spec/core/inlines-spec.js

```diff
@@ -224,6 +224,36 @@ describe("Core - Inlines", () => {
     expect(badOne.textContent).toBe("[[[#does-not-exist]]]");
   });
 
+  it("processes normative and informative `[[[spec/path#fragment]]]` into a link with the correct data-cite and text content", async () => {
+    const body = `
+      <p id="output">
+        [[[!HTML/path.html#some-normative-fragment]]]
```

Comment:
Lets use a real world example maybe?
```suggestion
        [[[!HTML/webstorage.html#user-tracking]]]
```